src/sentry/tasks/release_summary.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models import Activity, Release, ReleaseCommit
+from sentry.notifications.notifications.activity.release_summary import (
+    ReleaseSummaryActivityNotification,
+)
+from sentry.notifications.utils.participants import _get_release_committers
+from sentry.tasks.base import instrumented_task
+from sentry.types.activity import ActivityType
+
+
+@instrumented_task(name="sentry.tasks.digest.release_summary")
+def prepare_release_summary():
+    """
+    Summarize new issues in a release an hour after deployment
+    """
+    now = timezone.now()
+    start = now - timedelta(hours=1, minutes=5)
+    end = now - timedelta(hours=1)
+    releases = Release.objects.filter(
+        deploy__date_finished__gte=start,
+        deploy__date_finished__lt=end,
+        deploy__notified=True,
+    )
+
```

Comment:
Probably me being overly cautious, but do we need to persist this summary was sent already? I know the task runs every 5 minutes, and the `start` and `end` times should cover it. Just a thought. 

For this test, we might not need it?